shared/actions/search.js

```diff
@@ -31,50 +32,47 @@ type RawResult = {
   }
 }
 
-function parseFullName (rr: RawResult): string {
```

Comment:
This function is pulling the name from two different data sources so you
can't tell what your going to get. We explicitly want one or the other so
this fallback doesn't match our logic

On Thursday, July 14, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/actions/search.js
> <https://github.com/keybase/client/pull/3475#discussion_r70889081>:
>
> > @@ -31,50 +32,47 @@ type RawResult = {
> >    }
> >  }
> >
> > -function parseFullName (rr: RawResult): string {
>
> I like this better, you're duplicating login on how to read full_name,
> this will give you the full_name in one spot, what's the reasoning to
> remove this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3475/files/90050f8d710de01d3a7d9e9d0cd9a7084f1beb91#r70889081>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAfhUw7fWMneFljGWmQC0U29SG5qz5Afks5qVqw4gaJpZM4JM3tc>
> .
>